cluster/join.sls

```diff
@@ -1,5 +1,7 @@
 {%- from "cluster/map.jinja" import cluster with context -%}
 
+{% set lock_file = '/var/tmp/crmsh.lock' %}
```

Comment:
can we add more identity to this lock file perhaps? 

e.g
``` /var/tmp/crmsh_cluster_join_automation.lock```` or something like that so that we have an ID of which application is using it. 

Like that I would think `crmsh` is creating it